frontend/src/components/map/map.js

```diff
@@ -46,7 +46,12 @@ class Map extends Component {
     this.setState({ selectedCoords: null });
   };
 
+  //Set selected coords when user clicks on open space of map
   onMapClick = (coord) => {
+    if (!this.props.isAuthenticated) {
```

Comment:
The map never re-renders if the user is not logged in because the `selectedCoords` are never changed.